### ISSUE TITLE
Use a sensor for the lat and longitude

### ISSUE THREAD

**Issue by robbinonline** [Makin-Things/weather-radar-card#1]
Great card, it would be nice to use a sensor for the let and longitude instead typing the numbers.
For example a have a sensor: sensor.next_gp_let and sensor.next_gp_long and use it like this:

```
type: custom:weather-radar-card
marker_latitude: sensor.next_gp_let
marker_longitude: sensor.next_gp_long

```

**Comment by Makin-Things** [comment]
I am guessing you are looking at having the card follow your location? I will wait a little while to gather other enhancement requests so they can be prioritised before committing to doing it.
Also as a brand new card (although the code has been tested with the Australian version) I am going to prioritise any bugs found first.

**Comment by robbinonline** [comment]
Yes by example, in my case it would like to show the radar at the current F1 Grand Prix.

**Comment by Makin-Things** [comment]
Are you able to share the config of the sensors you use for the GP location. It may be useful for testing when I get to looking at that.
At zoom 10 the card is about 40km square, you must have very small states. I am not sure if increasing will help much since the radar data doesn't appear to be much more granular than that, but yes it is possible to adjust. Unfortunately right now it looks like you have perfectly fine weather so it is hard for me to observe.

**Comment by Makin-Things** [comment]
@robbinonline  I am considering doing this, but I need an example of a sensor config for the location.

**Comment by vividfog** [comment]
Here’s an example from Tesla integration. The location tracker has lat/long as attributes. 

Would be handy to have a second marker on the map, defined as a sensor. From which the lat/long attributes would be pulled. 

![IMG_2890](https://user-images.githubusercontent.com/75913791/229265207-2c9f9b2d-e994-47cc-a992-1a586dabbc84.jpeg)

**Comment by vividfog** [comment]
HA companion app location OTOH shows one state attribute with two values inside. This format would work for people who want to see the rain situation in “where I am” == where a HA tracked device is. Arguably this is a format with more users than the Tesla tracker format. And this would effectively work for the “weather along my route” scenario too: often it’s the HA user driving, with their companion app. So that’s where the car is. And that’s where to center the map and the marker (of choice?)

hope this helps 

![IMG_2892](https://user-images.githubusercontent.com/75913791/229265459-c754e827-bfa5-4ec1-85c9-be98c76e6f4e.jpeg)

**Comment by rv10guy** [comment]
Any update on this? It would be great to set the marker location via sensor data as well.

**Comment by jpettitt** [comment]
I've just started as maintainer - I'm closing all old issues as obsolete.  If this is still important to you please re-open.
